src/styles/main.scss

```diff
@@ -0,0 +1,147 @@
+$body-font: roboto, sans-serif;
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$white: #fff;
+$concrete-color: #f3f3f3;
+
+body {
+  margin: 0;
+  font-family: $body-font;
+}
+
+.card {
```

Comment:
according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) every BEM block should have 1 file - please create card file and move that piece of code there